service/urls.py

```diff
@@ -23,4 +23,6 @@
     url(r'^posts/(?P<pk>[0-9a-z\\-]+)/', service.posts.views.AllPostsViewSet.as_view({"get": "retrieve"}),
         name='post-detail'),
     url(r'^author/(?P<pk>[0-9a-z\\-]+)/posts/', service.posts.views.AuthorPostsList.as_view(), name='author-posts-list'),
+    url(r'^author/(?P<me>[0-9a-z\\-]+)/friends/(?P<you>[0-9a-z\\-]+)',
```

Comment:
The other host name is entirely superfluous (because if we don't know about the remote author, we just say no, because if they were friends, we would have got a friend request from them previously and thus knew about them), so it would be nice if the simpler URL was the way to go. However, the example given then uses the URL with the other host name in it.